apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-env browser */
+/* global TomSelect, MathJax */
+
+window.PLMultipleChoice = function (uuid) {
```

Comment:
Really nit, but should this be `PLMultipleChoiceSelect`? On the off chance that we have a new variation of multiple choice in the future?

Reply:
I had the same thought, but I will treat this as a nit and ignore it. It is entirely trivial to rename this in the future if we need to.